### PR TITLE
`First Steps` chapter rewritten and rephrased.

### PR DESCRIPTION
**Pull request by wargio**
Rewriting of the book `First Steps` chapter to update it removing wrong and useless info.

This PR also fixes the workflow and updates the assets included within the repo

### CODE REVIEW

File: src/first_steps/intro.md [hunk 1]
```diff
@@ -1,78 +1,119 @@
...
 | a[?]                  # Analysis commands
 ...
-| p[cdghtm?]            # Print commands
+| p[?]                  # Print commands
 ...
 
 ```
 
-**Note**: you can prepend a `?` after any command to see it's help.
+**Note**: You have the option to add a `?` before any command to access its help.
```
Comment (wargio) [inline src/first_steps/intro.md:18]: maybe we should mention about `??` which shows more info and examples.

File: src/first_steps/overview.md [hunk 2]
```diff
@@ -40,29 +81,31 @@ $ echo 'push eax;nop;nop' | rz-asm -f -
...
 ### Examples
 ```
-$ rz-hash file
+$ rz-hash file.bin
 file: 0x00000000-0x00000007 sha256: 887cfbd0d44aaff69f7bdbedebd282ec96191cce9d7fa7336298a18efc3c7a5a
 ```
 ```
-$ rz-hash -a md5 file
+$ rz-hash -a md5 file.bin
 file: 0x00000000-0x00000007 md5: d1833805515fc34b46c2b9de553f599d
```
Comment (wargio) [inline src/first_steps/overview.md:93]: this should be `file.bin`

File: src/first_steps/compilation_portability.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ## Compilation and Portability
 
-Currently the core of rizin can be compiled on many systems and architectures, but the main development is done on GNU/Linux with GCC, and on MacOS X with clang. Rizin is also known to compile on many different systems and architectures (including TCC and SunStudio).
+The core of Rizin can be compiled on various systems and architectures, with primary development conducted on GNU/Linux using GCC and on MacOS X with clang. Rizin is known to compile on a wide range of systems and architectures, including compatibility with TCC and SunStudio.
```
Comment (XVilka) [inline src/first_steps/compilation_portability.md:3]: s/The core of Rizin/Rizin. Also remove "conducted on Linux/macOS" and replace with "conducted on UNIX systems (Linux, macOS, etc)" or something like that.

Comment (XVilka) [reply src/first_steps/compilation_portability.md:3]: TCC (TinyCC), maybe a link to their repo.

Comment (XVilka) [reply src/first_steps/compilation_portability.md:3]: Also, I am not sure about SunStudio. Also Oracle uses Clang nowadays, I think. Just remove its mention completely.

File: src/first_steps/compilation_portability.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
-People often want to use rizin as a debugger for reverse engineering. Currently, the debugger layer can be used on Windows, GNU/Linux (Intel x86 and x86_64, MIPS, and ARM), OS X, FreeBSD, NetBSD, and OpenBSD (Intel x86 and x86_64)..
+For those interested in utilizing Rizin as a debugger for reverse engineering, the debugger layer is currently available on Windows, GNU/Linux (Intel x86 and x86_64, MIPS, and ARM), OS X, FreeBSD, NetBSD, and OpenBSD (Intel x86 and x86_64).
```
Comment (XVilka) [inline src/first_steps/compilation_portability.md:5]: s/OS X/macOS/.

File: src/first_steps/compilation_portability.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 
-Compared to core, the debugger feature is more restrictive portability-wise. If the debugger has not been ported to your favorite platform, you can disable the debugger layer with the meson option `-Ddebugger=false`. Have a look at [BUILDING.md](https://github.com/rizinorg/rizin/blob/dev/BUILDING.md) for more information.
+However, it's important to note that the debugger feature has more restrictive portability compared to the core. If the debugger hasn't been ported to your preferred platform, you have the option to disable the debugger layer using the meson option `-Ddebugger=false`. Refer to [BUILDING.md](https://github.com/rizinorg/rizin/blob/dev/BUILDING.md) for additional details.
 
-Note that there are I/O plugins that use GDB, WinDbg, or Wine as back-ends, and therefore rely on presence of corresponding third-party tools (in case of remote debugging - just on the target machine).
+Keep in mind that there are I/O plugins utilizing GDB, WinDbg, or Wine as back-ends, and they depend on the presence of corresponding third-party tools, especially in the case of remote debugging (on the target machine).
```
Comment (XVilka) [inline src/first_steps/compilation_portability.md:9]: WinDbg and KD

File: src/first_steps/intro.md [hunk 6]
```diff
@@ -1,78 +1,119 @@
 ## Basic Rizin Usage
 
-The learning curve is usually somewhat steep at the beginning. Although after an hour of using it you should easily understand how most things work, and how to combine the various tools rizin offers.
+The initial learning curve tends to be relatively steep. However, within an hour of usage, you should grasp the workings of most features and learn how to effectively utilize the different tools provided by `rizin`.
 
-To get a first overview over the basic commands just run Rizin and type `?`:
+For a quick introduction to basic commands, execute `rizin` on the command line and enter `?` to obtain an overview.
```
Comment (XVilka) [inline src/first_steps/intro.md:5]: No, suggest running `help` first. Then, `?` for more details

File: src/first_steps/intro.md (hunk 1 above)
Comment (XVilka) [reply src/first_steps/intro.md:18]: Definitely

File: src/first_steps/intro.md [hunk 7]
```diff
@@ -1,78 +1,119 @@
...
 
 ![learning_curve](learning_curve.png)
 
 ### Navigate and print
 
-Navigation, inspection and modification of a loaded binary file is performed using three simple actions: seek (to position), print (buffer), and alternate (write, append).
+Navigation, inspection, and modification of a loaded binary file involve three straightforward actions: seek (to position), print (buffer), and alternate (write, append).
 
-The 'seek' command is abbreviated as `s` and accepts an expression as its argument. The expression can be something like `10`, `+0x25`, or `[0x100+ptr_table]`. If you are working with block-based files, you may prefer to set the block size to a required value with `b` command, and seek forward or backwards with positions aligned to it. Use `s++` and `s--` commands to navigate this way.
+The 'seek' command is abbreviated as `s` and takes an expression as its argument. The expression can be something like `10`, `+0x25`, or `[0x100+ptr_table]`. When dealing with block-based files, you might prefer setting the block size to a required value with the `b` command and navigating forward or backward with positions aligned to it using `s++` and `s--` commands.
```
Comment (XVilka) [inline src/first_steps/intro.md:92]: These command do not exist anymore, use `sd` instead

File: src/first_steps/intro.md [hunk 8]
```diff
@@ -1,78 +1,119 @@
...
+> wx 90 90 90 90        # hexpairs
+> wa jmp 0x8048140      # assemble
+> wf inline.bin         # write contents of file
 ```
 
-Appending a `?` to a command will show its help message, for example, `p?`.
-Appending `?*` will show commands starting with the given string, e.g. `p?*`.
+### Visual mode and panels
+
+Rizin also offers visual mode via `V` and visual panels via `v`.
```
Comment (XVilka) [inline src/first_steps/intro.md:111]: Add also mention of the `Vv` mode, with function roster

**Review by XVilka — CHANGES_REQUESTED** [review]

File: src/plugins/rz-pm.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Creating an rz-pm package of the plugin
 
-As you remember rizin has its own [packaging manager](/tools/rz-pm/intro.md) and we can easily
+As you remember rizin has its own [packaging manager](../tools/rz-pm/intro.md) and we can easily
 add newly written plugin for everyone to access.
 
 All packages are located in [rizin-pm](https://github.com/rizinorg/rizin-pm) repository, and have
```
Comment (XVilka) [inline src/plugins/rz-pm.md:6]: This one should be rewritten since `rz-pm` now has different format and options.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: src/plugins/rz-pm.md (hunk 9 above)
Comment (wargio) [reply src/plugins/rz-pm.md:6]: this is just to fix a warning of missing file while compiling the book. once i will reach the `tools` folder i will.

**Review by XVilka — APPROVED** [review]

**Review by ITAYC0HEN — APPROVED** [review]
Oh wow! Great additions and fixes!

**Pull request merged**